MultiShop/base/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.utils.translation import gettext_lazy as _
+from django.utils.crypto import get_random_string
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+class User(AbstractUser):
+    email = models.EmailField(_('email address'), unique=True)
+    father_name = models.CharField(max_length=50)
+    GENDER_CHOICES = (
+        ('Female', 'Female'),
+        ('Male', 'Male'),
+        ('Other', 'Other'),
+    )
+    gender = models.CharField(max_length=10, choices=GENDER_CHOICES)
+    PROVINCE_CHOICES = (
+        ('Punjab', 'Punjab'),
+        ('Sindh', 'Sindh'),
+        ('KPK', 'KPK'),
+        ('Balochistan', 'Balochistan'),
+        ('Gilgit Baltistan', 'Gilgit Baltistan'),
+    )
+    province = models.CharField(max_length=50, choices=PROVINCE_CHOICES)
+    city = models.CharField(max_length=50)
+    dob = models.DateField(null=True, blank=True)
+    phone = models.CharField(max_length=20)
+    about = models.TextField(max_length=500)
+    profile_pic = models.ImageField(upload_to='profile_pic/')
+    updated = models.DateTimeField(auto_now=True)
+    created = models.DateTimeField(auto_now_add=True)
+    is_verified = models.BooleanField(default=False)
+    verification_token = models.CharField(max_length=100, blank=True)
+
```

Comment:
Add created_at, updated_at fields as well. Next time in every model must add these two fields. These should be auto date time populated.